### PR TITLE
SheltersScreen with button to Map and return button

### PR DESCRIPTION
**Pull request by trissve**
:rocket: roket

### CODE REVIEW

**Comment by trissve** [comment]
https://user-images.githubusercontent.com/86602714/236180391-bffab8b5-d949-43ad-a262-0b7310f426dc.mp4

File: MeowdoptMeApp/src/SheltersScreen.tsx [hunk 1]
```diff
@@ -1,24 +1,88 @@
 import * as React from 'react';
-import {View, Text, StyleSheet} from 'react-native';
+import {useState, useContext} from 'react';
+import {ScreenContext} from './Context';
```
Comment (tjzel) [inline MeowdoptMeApp/src/SheltersScreen.tsx:3]: Please remove unnecessary imports.

File: MeowdoptMeApp/src/SheltersScreen.tsx [hunk 2]
```diff
@@ -1,24 +1,88 @@
...
+import {View, Text, StyleSheet, Pressable} from 'react-native';
 
 function SheltersScreen() {
+  const [page, setPage] = useState(0);   // 0 to sheltersList, 1 - mapa
```
Comment (tjzel) [inline MeowdoptMeApp/src/SheltersScreen.tsx:7]: Making it a numerical value and then add a comment is not the best option here - code should be as self explanatory as possible. Consider making the state a string, for eg. `const [page, setPage] = useState('list')` or making it a boolean value: `const [isSheltersList, setIsSheltersList] = useState(true)`.

**Review by tjzel — CHANGES_REQUESTED** [review]

**Comment by trissve** [comment]
https://user-images.githubusercontent.com/86602714/236618790-69d0a783-98e1-4f3c-82ed-c57e8aca1f09.mp4

**Review by tjzel — APPROVED** [review]
:clap:

**Pull request merged**